testHelperFixtures/build.gradle

```diff
@@ -24,6 +24,9 @@ plugins {
     id 'java-test-fixtures'
 }
 
+java.sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
Why have these, specifically, set to JDK 11 when everything else is moving to JDK 17

Reply:
Since the proxy may be installed alongside other software (like the ES coordinator notes), the proxy is the one component that may NOT be deployed on a new host.  ES6.5 and later require Java 11, so if we were going to install on those hosts, we could use the same java runtime.  Users of older versions would have more friction - but this is a tradeoff between ease of development vs expected customer friction.